sdk/eventhub/event-hubs/package.json

```diff
@@ -106,7 +106,7 @@
   "devDependencies": {
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
-    "@azure/identity": "^1.1.0",
+    "@azure/identity": "2.0.0-beta.3",
```

Comment:
Wait, why does event hubs get beta.3 and everyone else gets beta.4? I feel left behind :smile:

Reply:
Now it should be on par with everyone 🥇